install/kubernetes/istio-rbac-beta.yaml

```diff
@@ -1,3 +1,5 @@
+# GENERATED FILE. Use with Kubernetes 1.7+
```

Comment:
cool ! thx!

Reply:
I think that the comments will be removed if I generate this file again? How about updating the README here a bit to reflect this?

Reply:
No, they will not be removed.
I will remove this file in a separate commit, it's no longer useful.

Reply:
I see, thanks @andraxylia